### ISSUE TITLE
[Feature Requests] Additional key binding suggestions

### ISSUE THREAD

**Issue by Anzan** [timhor/obsidian-editor-shortcuts#63]
As a Sublime Text user, here are some other key binding suggestions:

Code Editor Shortcut: Expand selection to brackets
Ctrl + Shift + M
Code Editor Shortcut: Insert cursor above
Ctrl + Alt + (up arrow)
Code Editor Shortcut: Insert cursor below
Ctrl + Alt + (down arrow)

Command palette: Open command palette
Ctrl + Shift + P
Quick switcher: Open quick switcher
Ctrl + P

This one isn't related to Sublime Text but just something I added personally:

Code Editor Shortcut: Expand selection to quotes
Ctrl + '

**Comment by timhor** [comment]
To clarify, are you suggesting these as _default_ key bindings? Since you can currently customise the key bindings for these actions via Obsidian's hotkey settings already.

**Comment by Anzan** [comment]
Yeah, just suggesting additional default key bindings for your plugin that corresponds to defaults in Sublime Text, I prefer to install your plugin with every new vault than having to set them up manually every time, which is what I'm currently doing.

**Comment by timhor** [comment]
Makes sense! I've been meaning to add some default key bindings users can optionally enable / switch between in settings (see [this card](https://github.com/timhor/obsidian-editor-shortcuts/projects/1#card-76888274)).

As a workaround for now, you can copy over the `hotkeys.json` from your existing vault's `.obsidian` directory.

**Comment by Anzan** [comment]
I'm a new user with Obsidian and yeah, I just figured the `hotkeys.json` trick, still, I think other Sublime Text users might enjoy as many familiar keys as possible. If you're trying to stick with keys that are the same in both VScode and ST, it might be too restrictive or too much of a compromise. Thanks for your plugin!

**Comment by timhor** [comment]
Yeah definitely agree, the new setting should solve this exact issue – I just need to find the time to build it in 😀

**Comment by Mikestriken** [comment]
Please add my request in #81 
(not sure if this is supposed to be some kind of "main feature request thread")
